sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -26,7 +29,7 @@ This package's
 and 
 [samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration) 
 
-## Version 1.0.0-preview.5 (2019-10-11)
+## 1.0.0-preview.5
```

Comment:
I note that further down this PR you include the dates, whereas here you remove them. Is there an inconsistency here?

Reply:
I have updated that file.